code/game/objects/objs.dm

```diff
@@ -179,6 +180,23 @@ var/global/list/reagents_to_log = list(FUEL, PLASMA, PACID, SACID, AMUTATIONTOXI
 	to_chat(M, "You eject \the [integratedpai] from \the [src].")
 	M.put_in_hands(eject_integratedpai_if_present())
 	playsound(src, 'sound/misc/cartridge_out.ogg', 25)
+	
+/obj/proc/configure_pai()
```

Comment:
This seems really annoying to use. And also if you can't touch the pAI, wirejack is useless.

Reply:
It's essentially the same as the PDA option, that's why I did it like this. I suppose I could try to make an action button for it or something, though that'd clutter the action bar.
Ninjas really don't need wirejack, considering they can cut open walls and doors.

Reply:
That's true, but I'm wondering if we're going to end up with a bunch of pAI who waste RAM.

Here's a hot take, what if we set ninja pai ram to 0 and give them every app except wirejack for free. Just edit the default software list.

Reply:
I was considering doing something like that, but in a seperate PR.